App.tsx

```diff
@@ -8,7 +8,7 @@
  * @format
  */
 
-import React from 'react';
```

Comment:
Change from "react" -> 'react'.

Reply:
I usually use single quotes but whenever i save my code, the double quotes are automatically added. It's an extension that does it for me.

Reply:
Change your tool to use single quote.